randrctl/ctl.py

```diff
@@ -201,12 +201,21 @@ def _configs(self):
                 yield config
 
     def get_randrctl(self):
-        config = ConfigParser(allow_no_value=True, strict=False)
-
         config_files = list(self._configs())
         config_files.reverse()  # reverse, so config file from the preferred home comes last (has highest precedence)
-        read = config.read(config_files)
-        logger.debug("read configuration from %s", read)
+
+        # Empty container to hold the configuration
+        config = {}
+        # Iterate over the list of config files
+        for config_file in config_files:
```

Comment:
Although `ConfigParser` had `strict=False` and parsed 2 configs, I think any sort of merge is counterintuitive. It should be either system-wide config (which should be deprecated eventually I believe), or user config. Never both.

Behavior is even harmful. Imagine, system config defines prior and post hooks to STOP and CONT i3, then user defines post hook to do something else, so system hook becomes overridden by users config and i3 receives SIG_STOP but never SIG_CONT.

Do you agree, that having only one config at a time would be simpler, cleaner and just better?

If yes, would you be willing to fix this part in your PR :) ?

Reply:
I was just trying to mimic how ConfigParser was being used, but I totally agree with your reasoning

In short, randrctl should load the system-wide config, but prefer the user's config (if available) 
And favour YAML in both cases, outputting warning messages if using INI. 
Did I get it right? If so, I can make the changes either today or tomorrow

Reply:
Yes, this is correct (just to clarify: system config may be also not available).

Looking forward for the changes! Thank you.